_episodes/03-create.md

```diff
@@ -346,8 +346,7 @@ quotes.txt
 
 One must be careful when specifying the target file name, since `mv` will
 silently overwrite any existing file with the same name, which could
-lead to data loss. An additional option, `mv -i` (or `mv --interactive`),
-can be used to make `mv` ask you for confirmation before overwriting.
+lead to data loss. By default, `mv` will not ask for confirmation before overwriting files. However, an additional option, `mv -i` (or `mv --interactive`), will cause `mv` to request such confirmation.
```

Comment:
Please split this long line - it will fail the automatic checks. Use the surrounding text as a guide for how long the lines should be.

Reply:
OK, I will fix it.